httpfetch/httpfetch.go

```diff
@@ -240,6 +257,15 @@ func (h *HTTPFetch) fetch() error {
 				slog.String("If-Modified-Since", prevCache.LastModified))
 			req.Header.Set("If-Modified-Since", prevCache.LastModified)
 		}
+
+		// If there is a frequency for this feed AND the time has not yet
+		// been reached then we terminate early
+		if time.Since(prevCache.Updated) < h.frequency {
+			h.logger.Debug("avoiding this fetch as the last time %s the feed was updated is less than %d ago",
```

Comment:
maybe changing the message to

> avoiding this fetch as the last time %s the feed was **checked** is less than %d ago

is less confusing? At first, I thought that "updated" refers to a new item found on that feed.

But that might just be me not being used to the new feature.

Reply:
Or even better 

> avoiding this fetch as the last time %s the feed was **fetched** is less than %d ago

which is coherent to what you set as description of the _frequency_ option

> The "frequency" argument tracks the time between the last fetch of a feed, and
> allows you to say "Fetch this feed only if it was fetched >XX minutes ago", even
> if the sleep-delay is set to something short like a minute.